newm-server/src/main/kotlin/io/newm/server/features/user/repo/UserRepositoryImpl.kt

```diff
@@ -336,6 +341,12 @@ internal class UserRepositoryImpl(
         transaction {
             getUserEntityByEmail(email).passwordHash = passwordHash
         }
+        emailRepository.send(
+            to = email,
+            subject = environment.getConfigString("passwordReset.subject"),
+            messageUrl = environment.getConfigString("passwordReset.messageUrl"),
+            messageArgs = mapOf("owner" to user.stageOrFullName)
```

Comment:
The `user` argument passed in this function call only has the `email` and `password` fields populated. So, `user.stageOrFullName` won't work. Instead, you will need to get it from the database. Since, we're already looking up in the database at line 342, you could do reuse the transaction by doing something like this:
```
val userEntity = transaction {
    getUserEntityByEmail(email).apply {
        this.passwordHash = passwordHash
    }
}
emailRepository.send(
    to = email,
    subject = environment.getConfigString("passwordReset.subject"),
    messageUrl = environment.getConfigString("passwordReset.messageUrl"),
    messageArgs = mapOf("owner" to userEntity.stageOrFullName)
)
```